File: mybatis-plus-join-test/test-join/src/main/java/com/github/yulichang/test/join/service/impl/UserServiceImpl.java
package com.github.yulichang.test.join.service.impl;

import com.github.yulichang.base.MPJBaseServiceImpl;
import com.github.yulichang.test.join.entity.UserDO;
import com.github.yulichang.test.join.mapper.UserMapper;
import com.github.yulichang.test.join.service.UserService;
import org.springframework.stereotype.Service;

@Service
public class UserServiceImpl extends MPJBaseServiceImpl<UserMapper, UserDO> implements UserService {


}

File: mybatis-plus-join-test/test-kotlin/src/main/java/com/github/yulichang/test/kt/service/impl/UserServiceImpl.kt
package com.github.yulichang.test.kt.service.impl

import com.github.yulichang.base.MPJBaseServiceImpl
import com.github.yulichang.test.kt.entity.UserDO
import com.github.yulichang.test.kt.mapper.UserMapper
import com.github.yulichang.test.kt.service.UserService
import org.springframework.stereotype.Service

@Service
open class UserServiceImpl : MPJBaseServiceImpl<UserMapper?, UserDO?>(), UserService
